docs/developer/payments/transactions.mdx

```diff
@@ -450,6 +450,80 @@ Once `Checkout` is converted to into `Order`, the `authorizeStatus` might change
 - **For orders**: pending Transactions **are not** considered "paid" and will not be included in the [`Order.authorizeStatus`](/api-reference/orders/enums/order-authorize-status-enum.mdx#values) calculation.
 :::
 
+### TransactionItem and  TransactionEvents pspReference Matching
+
+Both [TransactionItem](/api-reference/payments/objects/transaction-item)
+and [TransactionEvent](/api-reference/payments/objects/transaction-event)
+contain a `pspReference` field, which refers to the payment service provider reference.
+	-	The `pspReference` on `TransactionItem` identifies the overall payment.
+	-	The `pspReference` on `TransactionEvent` identifies specific actions that occurred within the `Transaction`.
+
+**Key Assumptions:**
+- Only one `TransactionItem` instance can exist with a given `pspReference`.
+- For a specific `TransactionItem`, only one `TransactionEvent` of a given `type` and `pspReference` may exist, except for the following types:
+  -	`AUTHORIZATION_ACTION_REQUIRED`
+	-	`CHARGE_ACTION_REQUIRED`
+	-	`INFO`
+- Only one `TransactionEvent` of type `AUTHORIZATION_SUCCESS` is allowed to exist.
+
+#### Setting the `pspReference` on the `TransactionItem`
+
+The pspReference value in a TransactionItem may initially be empty if it is not provided
+during the TransactionCreate mutation or as a result of the TransactionInitialize mutation.
+When the first `TransactionEvent` containing a `pspReference` is reported,
+this value is automatically set on the associated `TransactionItem`.
+
+The `pspReference` can be updated through the `transactionUpdate` mutation if necessary.
+
+#### Events with optional `pspReference` 
+
+The following events do not require a `pspReference`:
+- `CHARGE_ACTION_REQUIRED`
+- `AUTHORIZATION_ACTION_REQUIRED`
+- `CHARGE_FAILURE`
+- `AUTHORIZATION_FAILURE`
+- `REFUND_FAILURE`
+- `CANCEL_FAILURE`
+
+If `pspReference` is missing, the event will be created as usual but will be excluded
+from transaction amount recalculations. As a result, it will not affect
```

Comment:
Maybe worth to mentioned, that missing PSPreference will not have an impact on existing transactionEvent with pspReference:
I mean the part that is described here:
![image](https://github.com/user-attachments/assets/aead59e8-dc8f-4c6e-b51d-09fc380b9062)
Ref: https://docs.saleor.io/developer/payments/lifecycle#authorization-events

Reply:
Done